core/ledger/kvledger/txmgmt/statedb/statecouchdb/statecouchdb.go

```diff
@@ -167,16 +171,22 @@ func (provider *VersionedDBProvider) HealthCheck(ctx context.Context) error {
 // VersionedDB implements VersionedDB interface
 type VersionedDB struct {
 	couchInstance      *couchInstance
-	metadataDB         *couchDatabase            // A database per channel to store metadata such as savepoint.
-	chainName          string                    // The name of the chain/channel.
-	namespaceDBs       map[string]*couchDatabase // One database per deployed chaincode.
-	committedDataCache *versionsCache            // Used as a local cache during bulk processing of a block.
+	metadataDB         *couchDatabase              // A database per channel to store metadata such as savepoint.
+	chainName          string                      // The name of the chain/channel.
+	namespaceDBs       map[string]*couchDatabase   // One database per namespace.
+	namespaceDBsInfo   map[string]*namespaceDBInfo // One namespaceDBInfo per namespace.
```

Comment:
```suggestion
	namespaceDBs       map[string]*couchDatabase   // One database per namespace.
	channelMetadata    *channelMetadata .
```
`namespaceDBs` and `namespaceDBsInfo` look confusing.